diff --git a/app/build.gradle b/app/build.gradle
index b902bfe..555d003 100755
--- a/app/build.gradle
+++ b/app/build.gradle
@@ -85,7 +85,7 @@ dependencies {
     compile 'com.taobao.android:weex_sdk:0.10.1.1'
     compile project(':appframework')
     compile project(':weexplugin')
-//    compile project(':plugin')
+    compile project(':plugin')
     compile 'com.google.code.findbugs:jsr305:2.0.1'
     compile 'com.squareup.okhttp:okhttp:2.3.0'
     compile 'com.squareup.okhttp:okhttp-ws:2.3.0'
diff --git a/settings.gradle b/settings.gradle
index 827f265..d91d926 100755
--- a/settings.gradle
+++ b/settings.gradle
@@ -1,8 +1,8 @@
 include ':app'
 include ':weexplugin'
 include ":appframework"
-//include ":plugin"
-//project(":plugin").projectDir=new File("../plugin")
+include ":plugin"
+project(":plugin").projectDir=new File("../plugin")
 
 //include ":inspector"
 //project(":inspector").projectDir=new File("../inspector")
